README.md

```diff
@@ -221,38 +234,38 @@ The table below shows the track parameters that can be adjusted along with defau
 
 ### index-logs-fixed-daily-volume
 
-This challenge indexes a fixed amount of logs per day into daily indices. The table below shows the track parameters that can be adjusted along with default values:
+This challenge indexes a fixed (raw) logging volume of logs per day into daily indices. This challenge will complete tasks as quickly as possible and won't take the amount of days specified in the number_of_days field.  The table below shows the track parameters that can be adjusted along with default values:
 
 | Parameter               | Explanation                                                                                                                            | Type  | Default Value |
 | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------- | ----- | ------------- |
 | `bulk_indexing_clients` | Number of bulk indexing clients/connections                                                                                            | `int` | `8`           |
 | `daily_logging_volume`  | The raw logging volume. Supported units are bytes (without any unit), `kb`, `MB` and `GB`). For the value, only integers are allowed.  | `str` | `100GB`       |
-| `number_of_days`        | The number of days for which data should be generated.                                                                                 | `int` | `24`          |
+| `number_of_days`        | The number of simulated days for which data should be generated.                                                                                 | `int` | `24`          |
 | `shard_count`           | Number of primary shards                                                                                                               | `int` | `3`           |
 
 ### index-and-query-logs-fixed-daily-volume
 
-Indexes several days of logs with a fixed (raw) logging volume per day and running queries concurrently. The table below shows the track parameters that can be adjusted along with default values:
+Indexes several days of logs with a fixed (raw) logging volume per day and running queries concurrently. This challenge will complete tasks as quickly as possible and won't take the amount of days specified in the number_of_days field.  The table below shows the track parameters that can be adjusted along with default values:
```

Comment:
nit: there are two whitespaces between `field.` and `The table`.

Reply:
Arg that's going to be a tough one to kick